enarx-keep-sev-shim/Cargo.toml

```diff
@@ -33,3 +33,7 @@ panic = "abort"
 codegen-units = 1
 incremental = false
 lto = true
+
+[features]
+default = []
+debug = []
```

Comment:
Why a feature? Why not use `#[cfg(debug_assertions)]` or `#[cfg(not(debug_assertions))]`?

Reply:
Because for the integration tests the actual output is tested, so every additional output will fail the integration tests.

Reply:
@haraldh I think that just raises the question: should our test suite run in release mode? I rather like the idea that our CI tests the bits we would ship.

Reply:
@npmccallum it should run in `--debug` and `--release` to catch all the `debug_assert!()`